src/components/GreetingCard.jsx

```diff
@@ -32,6 +44,7 @@ export default function GreetingCard() {
 
   return (
     <div css={cardStyles}>
+      <img css={logoStyle} src={logoSrc} alt={logoAltText}></img>
```

Comment:
Image tags are self closing `<img \>` instead of `<img></img>` since the tag doesn't accept children. I doubt it will cause a bug, but for future reference.

Reply:
Updated!